src/main/java/uk/gov/hmcts/reform/sscs/services/CaseLoaderService.java

```diff
@@ -57,8 +57,8 @@ public void process() {
             log.debug("*** case-loader *** file validated successfully: {}", file.getName());
             if (file.isDelta()) {
                 if (null == latestRef) {
-                    String message = String.format("No reference data processed for this delta: %s", file.getName());
-                    throw new TransformException(message);
+                    throw new TransformException(String.format("No reference data processed for this delta: %s",
```

Comment:
Do you need to log this error?

Reply:
Yes, it's good to know what delta file is failing.

Reply:
Still no logging so this won't show on app insights - you need a `LOG.error(message, exception);` Reference Notification Service